backend/src/models/variety.rs

```diff
@@ -0,0 +1,19 @@
+use crate::schema::varieties::{self, all_columns};
+use diesel::{Identifiable, QueryDsl, QueryResult, Queryable, RunQueryDsl};
+use serde::{Deserialize, Serialize};
+
+use crate::config::db::Connection;
+
+#[derive(Identifiable, Queryable, Serialize, Deserialize)]
+pub struct Varietie {
+    pub id: i32,
+    pub tags: Vec<Option<String>>,
+    pub species: String,
+    pub variety: Option<String>,
+}
+
+impl Varietie {
```

Comment:
```suggestion
impl Variety {
```

Reply:
I'll have to look into this because Diesel automatically searches for the table with plural form of the model but only in its basic form: Seed -> Seeds, Variety -> Varieties (doesn't work out of the box), there should be a way to define this in diesel